src/operators.jl

```diff
@@ -449,23 +452,54 @@ function _mul!(ret::AbstractVecOrMat{<:GenericAffOrQuadExpr},
 end
 
 function _mul!(ret::AbstractMatrix{<:GenericAffOrQuadExpr},
-               A::AbstractMatrix, B::SparseMatrixCSC)
-    _fill_with_zeros!(ret)
+               A::AbstractMatrix, B::SparseMatrixCSC,
+               α_expr=one(eltype(ret)),
+               β=zero(eltype(ret)))
```

Comment:
When are these called with anything other than the defaults?

Reply:
I can't find anywhere it's called with values different from alpha=one and beta=zero in `SparseArrays/src/linalg.jl`. I don't know where it's documented either